policyengine_us/parameters/gov/contrib/states/ny/wftc/amount/max.yaml

```diff
@@ -0,0 +1,19 @@
+description: New York provides the following Working Families Tax Credit amount.
+metadata:
+  unit: currency-USD
+  period: year
+  uprating:
+    parameter: gov.irs.uprating
+    rounding:
+      type: downwards
+      interval: 50
+  label: New York Working Families Tax Credit amount
+  reference: 
+    - title: Instructions for Mississippi State FORMS 80-105 AND 80-205 (2)(A)
```

Comment:
hmm

Reply:
Will even out the formatting in a bit, still missing some rules